qemu/tests/timerdevice_clock_drift_with_sleep.py

```diff
@@ -38,36 +38,37 @@ def verify_elapsed_time():
         output = session.cmd_output(get_time_cmd, timeout=timeout)
 
         times_list = output.splitlines()[1:]
-        times_list = [_ for _ in times_list if _ > 10.0 or _ < 11.0]
+        times_list = [_ for _ in times_list if float(_) < 10.0 or float(_) > 11.0]
 
         if times_list:
-            raise error.TestFail("Unexpected time drift found:"
-                                 " Detail: '%s'" % output)
+            test.fail("Unexpected time drift found:"
+                      " Detail: '%s' \n timelist: %s" % (output, times_list))
 
-    error.context("Sync the host system time with ntp server", logging.info)
-    utils.system("yum install -y ntpdate; ntpdate clock.redhat.com")
+    error_context.context("Sync the host system time with ntp server", logging.info)
+    ntp_sync_cmd = "yum install -y ntpdate; ntpdate clock.redhat.com"
```

Comment:
can we use `utils_package` to handle package installation ? it is handled in better way.